src/network.js

```diff
@@ -321,6 +323,7 @@ export class BlockstackNetwork {
         }
       })
       .then((nameInfo) => {
+        console.log(`nameInfo: ${JSON.stringify(nameInfo)}`)
```

Comment:
this should also be a debug statement

Reply:
Thanks for catching this! I pushed a fixed.